packages/drawer/rc/utils.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-prototype-builtins */
+
+export const createActionCreator = type => dispatch => payload =>
+  dispatch({ payload, type })
```

Comment:
this partially implements the flux standard action pattern. we can expand with errors/meta/whatever as needed